secret-service/pkg/backend/secretbackend.go

```diff
@@ -12,4 +12,6 @@ type SecretBackend interface {
 	UpdateSecret(model.Secret) error
 	DeleteSecret(model.Secret) error
 	GetSecrets() ([]model.GetSecretResponseItem, error)
+
+	ScopeBackend
```

Comment:
This looks misplaced, but I guess it's valid Go Code...

I would however favour something like
```
type Backend interface {
    SecretBackend
    ScopeBackend
}
```